packages/app/src/components/VisualizationOptions/Options/Legend.js

```diff
@@ -81,7 +81,12 @@ const Legend = ({
                     <div>
                         <FieldSet>
                             <UiCoreLegend>
-                                <span className={tabSectionTitle.className}>
+                                <span
+                                    className={tabSectionTitle.className}
+                                    style={{
+                                        marginTop: hideStyleOptions ? 8 : 0,
```

Comment:
Use `spacers.dp8` instead of `8`?
Could all of this be put in a class that's used conditionally instead of using `style`?

Reply:
Refactored to use a class name with `cx` and a specific style for the additional margin.